CDCgov/prime-reportstream
# Modified Admin UI Organization check Page

This PR corrects Admin UI Organization check page.

Test Steps:
1. Pull the repos to your local host
2. cd frontend-react
3. yarn start:localdev
4. Login to local staging
5. Select a receiver org
6. Press check
Output:
Before:
![Before-Check-fix](https://github.com/CDCgov/prime-reportstream/assets/20068283/76a81b7c-077d-4760-9d58-ae93428957ff)

After:
![After-Check-fix](https://github.com/CDCgov/prime-reportstream/assets/20068283/cee523a2-1fe6-49a5-93f0-6f25a28bf2b9)


## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
